string                      source
float64                     startup_time
---
bool                        successful
watchdog_msgs/SystemStatus  status
